source/utilities.cc

```diff
@@ -2148,14 +2148,38 @@ namespace aspect
 
 
     std::string
-    expand_ASPECT_SOURCE_DIR (const std::string &location)
+    expand_paths (const std::string &location)
     {
+      // Check for environment variable override to ASPECT_SOURCE_DIR
+      char const *ASPECT_DATA_DIR_env = getenv("ASPECT_DATA_DIR");
+      if (ASPECT_DATA_DIR_env != NULL)
+        {
+          return Utilities::replace_in_string(location,
+                                              "$ASPECT_DATA_DIR",
+                                              ASPECT_DATA_DIR_env);
+        }
+      else
+      {
+        // Otherwise, use the default define from config.h
+        return Utilities::replace_in_string(location,
+                                            "$ASPECT_DATA_DIR",
+                                            ASPECT_DATA_DIR);
+      }
+
+      // Otherwise, use the default define from config.h
```

Comment:
this comment is out of date

Reply:
In which way? ASPECT_SOURCE_DIR is defined in config.h.